### DISCUSSION TITLE
Is it possible to specify a different Config directory apart from ~/.config

### DISCUSSION THREAD

**Discussion by thethoughtcode** [ferdium/ferdium-app#1286] [Q&A]
While `~/.config` directory is a reasonable place to put the config files, is it possible to change this location or maybe specify the configuration directory when Ferdium is started?

**Comment by vraravam** [comment]
What is it you are trying to do? If you mean to take a backup, you can always stop Ferdium, move that folder to the desired location, and create a symbolic link back to the location Ferdium expects (`~/.config`) and then restart Ferdium. I do that on my mac for the same exact reason.

**Comment by thethoughtcode** [comment]
Yes, along the same lines. But ability to specify a different folder could allow to run separate instances of the application at the same time. While Workspaces do allow to have clear separation, having a completely separate instances makes this distinction even more explicit.
